lib/livebook/hubs/enterprise_client.ex

```diff
@@ -25,24 +26,17 @@ defmodule Livebook.Hubs.EnterpriseClient do
     Server.send_request(GenServer.call(pid, :get_server), data)
   end
 
-  @doc """
-  Disconnects the Enterprise client with WebSocket server.
-  """
-  @spec disconnect(pid()) :: :ok
-  def disconnect(pid) do
-    GenServer.cast(pid, :disconnect)
-  end
-
   @doc """
   Subscribe to WebSocket Server events.
 
   ## Messages
 
-    * `{:unknown, :error, reason}`
     * `{:connect, :ok, :waiting_upgrade | :connected}`
```

Comment:
Do we still receive :waiting_upgrade event?

Reply:
I can remove, I'm still sending, but I don't see an use atm, it only helps on tests

Reply:
Let’s remove it then!